source/common/runtime/runtime_features.cc

```diff
@@ -45,6 +45,7 @@ RUNTIME_GUARD(envoy_reloadable_features_http1_lazy_read_disable);
 RUNTIME_GUARD(envoy_reloadable_features_http2_allow_capacity_increase_by_settings);
 RUNTIME_GUARD(envoy_reloadable_features_http2_new_codec_wrapper);
 RUNTIME_GUARD(envoy_reloadable_features_http_100_continue_case_insensitive);
+RUNTIME_GUARD(envoy_reloadable_features_http_local_ratelimit_match_all_descriptors);
```

Comment:
This guard isn't HTTP specific so I would do:

```suggestion
RUNTIME_GUARD(envoy_reloadable_features_local_ratelimit_match_all_descriptors);
```